tests/rkt_gc_test.go

```diff
@@ -45,32 +47,78 @@ func TestGC(t *testing.T) {
 	gcCmd := fmt.Sprintf("%s gc --mark-only=true --expire-prepared=0 --grace-period=0", ctx.Cmd())
 	spawnAndWaitOrFail(t, gcCmd, 0)
 
+	pods := podsRemaining(t, ctx)
+	if len(pods) == 0 {
+		t.Fatalf("pods should still be present in rkt's data directory")
+	}
+
+	gcCmd = fmt.Sprintf("%s gc --mark-only=false --expire-prepared=0 --grace-period=0", ctx.Cmd())
+	spawnAndWaitOrFail(t, gcCmd, 0)
+
+	pods = podsRemaining(t, ctx)
+	if len(pods) != 0 {
+		t.Fatalf("no pods should exist rkt's data directory, but found: %v", pods)
+	}
+}
+
+func podsRemaining(t *testing.T, ctx *testutils.RktRunCtx) []os.FileInfo {
 	gcDirs := []string{
 		filepath.Join(ctx.DataDir(), "pods", "exited-garbage"),
 		filepath.Join(ctx.DataDir(), "pods", "prepared"),
 		filepath.Join(ctx.DataDir(), "pods", "garbage"),
+		filepath.Join(ctx.DataDir(), "pods", "run"),
 	}
 
+	var remainingPods []os.FileInfo
 	for _, dir := range gcDirs {
 		pods, err := ioutil.ReadDir(dir)
 		if err != nil {
 			t.Fatalf("cannot read gc directory %q: %v", dir, err)
 		}
-		if len(pods) == 0 {
-			t.Fatalf("pods should still exist in directory %q", dir)
-		}
+		remainingPods = append(remainingPods, pods...)
 	}
 
-	gcCmd = fmt.Sprintf("%s gc --mark-only=false --expire-prepared=0 --grace-period=0", ctx.Cmd())
-	spawnAndWaitOrFail(t, gcCmd, 0)
+	return remainingPods
+}
 
-	for _, dir := range gcDirs {
-		pods, err := ioutil.ReadDir(dir)
-		if err != nil {
-			t.Fatalf("cannot read gc directory %q: %v", dir, err)
-		}
-		if len(pods) != 0 {
-			t.Fatalf("no pods should exist in directory %q, but found: %v", dir, pods)
-		}
+func TestGCAfterUnmount(t *testing.T) {
+	if !common.SupportsOverlay() {
+		t.Skip("Overlay fs not supported.")
+	}
+
+	ctx := testutils.NewRktRunCtx()
+	defer ctx.Cleanup()
+
+	imagePath := patchImportAndFetchHash("inspect-gc-test-run.aci", []string{"--exec=/inspect --print-msg=HELLO_API --exit-code=0"}, t, ctx)
```

Comment:
yep, delicious copypasta, it doesn't really matter but I can remove it from the statements